### PR TITLE
Added runOnlyThis option to ruleBuilder

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**

### CODE REVIEW

File: src/CLI/Config.php [hunk 1]
```diff
@@ -11,14 +11,32 @@ class Config
...
+        if ($this->runOnlyARule) {
+            return $this;
+        }
+
+        /** @var ArchRule $rule */
+        foreach ($rules as $rule) {
+            if ($rule->isRunOnlyThis()) {
+                $rules = [];
+                $rules[] = $rule;
+
```
Comment (fain182) [inline src/CLI/Config.php:34]: maybe it's simpler to insert the return here and remove the runOnlyARule attribute?

File: tests/Unit/CLI/ConfigTest.php [hunk 2]
```diff
@@ -31,4 +31,28 @@ public function test_it_should_create_config(): void
...
+            ->should(new HaveNameMatching('*Service'))
+            ->because('all services should be end name with Service')
+            ->runOnlyThis();
+
+        $config = new Config();
+        $config->add($classSet, ...[$rule1, $rule2]);
+
+        $this->assertInstanceOf(Config::class, $config);
+
+        $classSetRulesExpected[] = ClassSetRules::create($classSet, ...[$rule2]);
```
Comment (fain182) [inline tests/Unit/CLI/ConfigTest.php:55]: I suppose the spread operator of the one element array is a typo?

File: src/CLI/Config.php (hunk 1 above)
Comment (AlessandroMinoccheri) [reply src/CLI/Config.php:34]: You can call multiple times this method with different ClassSet, it's not the best approach I know but adding the `return` here I think it's not correct.
What do you think?

File: tests/Unit/CLI/ConfigTest.php (hunk 2 above)
Comment (AlessandroMinoccheri) [reply tests/Unit/CLI/ConfigTest.php:55]: I think it's correct, I copied the previous test and it seems to work fine.

File: src/CLI/Config.php (hunk 1 above)
Comment (fain182) [reply src/CLI/Config.php:34]: oh, i get it now... you are right!

File: tests/Unit/CLI/ConfigTest.php (hunk 2 above)
Comment (fain182) [reply tests/Unit/CLI/ConfigTest.php:55]: I was saying that:
`...[$rule2]` is the same as `$rule2`
but if you prefer to keep it like this for consistency with other tests it's ok, it's irrelevant
:slightly_smiling_face:

**Pull request merged**